modules/node_modules/@ncigdc/components/DownloadButton.js

```diff
@@ -28,14 +31,20 @@ const DownloadButton = ({
   filters = {},
   active,
   setActive,
+  altMessage = false,
+  style = {},
   extraParams = {},
+  dispatch,
 }) => {
-  const icon = <i key="icon" className={`fa fa-download ${active ? 'fa-spinner fa-pulse' : ''}`} />;
+  const icon = active ? <Spinner /> : <DownloadIcon />;
```

Comment:
I would personally prefer this

```js
const Icon = active ? Spinner : DownloadIcon;
```

Then instead of 
```js
{text || [icon, <Hidden key="hidden">download</Hidden>]}
```
Have
```js
{text || [<Icon key="icon"/>, <Hidden key="hidden">download</Hidden>]}
```

And instead of `leftIcon={text && icon}` have `leftIcon={text && <Icon />}`